robot/Cumulus/resources/NPSP.py

```diff
@@ -248,31 +248,31 @@ def select_related_row(self, value):
 #             else:
 #                 drop_down = npsp_lex_locators['rel_loc_dd'].format(index)
 #                 self.selenium.get_webelement(drop_down).click()
-#                 self.selenium.get_webelement(drop_down).click()      
-                
+#                 self.selenium.get_webelement(drop_down).click()
+
 
 
-    def click_id(self, title):  
+    def click_id(self, title):
         locator=npsp_lex_locators['aff_id'].format(title)
-        self.selenium.get_webelement(locator).click()     
-         
-    def select_object_dropdown(self): 
+        self.selenium.get_webelement(locator).click()
+
+    def select_object_dropdown(self):
         locator=npsp_lex_locators['object_dd']
-        self.selenium.get_webelement(locator).click()   
-        
+        self.selenium.get_webelement(locator).click()
+
     def check_status(self, acc_name):
         aff_list = npsp_lex_locators['aff_status'].format(acc_name)
-        aff_list_text=self.selenium.get_webelement(aff_list).text 
+        aff_list_text=self.selenium.get_webelement(aff_list).text
         self.aff_id=npsp_lex_locators['aff_id'].format(acc_name)
         self.aff_id_text=self.selenium.get_webelement(self.aff_id).text
-        return self.aff_id_text,aff_list_text     
-            
-            
+        return self.aff_id_text,aff_list_text
+
+
     def get_id(self):
         locator=npsp_lex_locators['click_aff_id'].format(self.aff_id_text)
-        self.selenium.get_webelement(locator).click()   
-        
-#     @capture_screenshot_on_error    
+        self.selenium.get_webelement(locator).click()
+
+#     @capture_screenshot_on_error
 #     def navigate_to_and_validate_field_value(self, field,status,value,section=None):
 #         """If status is 'contains' then the specified value should be present in the field
 #                         'does not contain' then the specified value should not be present in the field
```

Comment:
Is there a specific reason you commented this code but not delete it ?

Reply:
this was old code I made a copy and experimented with it but forgot to delete at that time, I will delete this